deploy/crds/hypercloud.tmaxanc.com_v1alpha1_virtualmachineimage_volume_cr.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: hypercloud.tmaxanc.com/v1alpha1
```

Comment:
does we need this file? or create one vmim_cr file with comment

Reply:
I think that it is not necessary if the cr yaml is for guide purposes.